src/parse/parse_mat.jl

```diff
@@ -29,44 +28,49 @@ function parse_mat(file::String; kwargs...)::Dict
 end
 
 function mat_to_exapf(data_mat)
-    
+
     # create new data structure
     data = Dict{String,Array}()
-    
+
     # baseMVA
     data["baseMVA"] = [data_mat["baseMVA"]]
 
     # buses
     BUS_I, BUS_TYPE, PD, QD, GS, BS, BUS_AREA, VM, VA, BASE_KV, ZONE, VMAX, VMIN,
     LAM_P, LAM_Q, MU_VMAX, MU_VMIN = IdxSet.idx_bus()
-    
```

Comment:
Thanks for the style fixes! Right now they don't come to me automatically but I'll promise I'll get better.

Reply:
My own code is not following exactly the style guide, I am afraid. Maybe at some point we would like to plug a formatter to autoformat the code.